contracts/src/test/7683/T1BasicSwapE2E.t.sol

```diff
@@ -804,7 +844,7 @@ contract T1BasicSwapE2E is BaseTest {
             "relayMessage(address,address,uint256,uint256,bytes)", from, to, msgValue, nonce, outerMessage
         );
 
-        rollup.setProofOfFill7683Root(batchIndex, keccak256(innerMessage));
+        rollup.setProofOfFill7683Root(batchIndex, proofOfFill7683Root);
```

Comment:
we could probably call `finalizeBatchWithProof` instead of `finalizeBundleWithProof` to get around using a custom setter